manifests/update.pp

```diff
@@ -88,10 +88,19 @@
   validate_bool($reboot_required)
 
   # Windows update service.
-  service { $service:
-    ensure => 'running',
-    enable => true,
-  }
+  case $::kernelmajversion {
+    '6.3', '10.0' : {
```

Comment:
When this was first written the idea was to make it maximally likely that updates are regularly checked for and downloaded. Since removing `ensure => running` entirely could have an impact on versions of Windows that do not have the "triggered start" feature, I slightly prefer the current version of the change over what `puppetlabs-wsus_client` is doing.